### PR TITLE
[plugin.video.orftvthek@leia] 0.12.4

### PR DESCRIPTION
**Pull request by s0faking**
### Add-on details:

- **General**
  - Add-on name: ORF TVthek
  - Add-on ID: plugin.video.orftvthek
  - Version number: 0.12.4
  - Kodi/repository version: leia

- **Code location**
  - URL: https://github.com/s0faking/plugin.video.orftvthek
  
ORF TVthek - This plugin provides access to the Austrian "ORF TVthek"

### Description of changes:

v0.12.4 (30/01/2022)
            [fix] fixed drm content (license server error 500)
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.orftvthek/resources/lib/Addon.py [hunk 1]
```diff
@@ -218,17 +216,18 @@ def run():
...
                     play_item.setProperty('inputstream.adaptive.stream_headers', headers)
                     play_item.setProperty('inputstream.adaptive.manifest_type', 'hls')
                 debugLog("Restart Stream Url: %s; play_item: %s" % (streaming_url, play_item))
-                xbmcplugin.setResolvedUrl(pluginhandle, True, listitem=play_item)
-                listCallback(False, pluginhandle)
-                #xbmc.Player().play(streaming_url, play_item)
+                #This works on matrix. On Kodi <19 the stream wont play
+                #xbmcplugin.setResolvedUrl(pluginhandle, True, listitem=play_item)
+                #listCallback(False, pluginhandle)
+                xbmc.Player().play(streaming_url, play_item)
```
Comment (basrieter) [inline plugin.video.orftvthek/resources/lib/Addon.py:230]: It should work and the `xbmc.Player().play()` should not be used. You are missing a `play_item.setProperty("IsPlayable", "true")`.

**Review by basrieter — DISMISSED** [review]
Please fix the "resolve" issue.

**Review by basrieter — CHANGES_REQUESTED** [review]
Please fix the "resolve" issue.

**Comment by s0faking** [comment]
> Please fix the "resolve" issue.

i tried it. this doesn't work. i'm getting "Attempt to use invalid handle -1" messages but the video won't play. i only need this in on the restart option for livestreams. i'm  not sure but i think this happens here because this restart is launched from the context-menu of a livestream (via RunPlugin()). enen92 already looked into this issue some time ago. not sure but i think he opened an issue on the kodi core for this.

**Comment by basrieter** [comment]
> > Please fix the "resolve" issue.
> 
> i tried it. this doesn't work. i'm getting "Attempt to use invalid handle -1" messages but the video won't play. i only need this in on the restart option for livestreams. i'm not sure but i think this happens here because this restart is launched from the context-menu of a livestream (via RunPlugin()). enen92 already looked into this issue some time ago. not sure but i think he opened an issue on the kodi core for this.

@enen92 can you confirm this?

**Comment by enen92** [comment]
Yeah that's right, playlist limitations on the API, we could well have a method for that in kodi

**Review by basrieter — APPROVED** [review]

**Pull request merged**